jpa-idempotent-repository/pom.xml

```diff
@@ -0,0 +1,396 @@
+					<artifactId>quarkus-maven-plugin</artifactId>
+					<version>${quarkus.platform.version}</version>
+				</plugin>
+
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-failsafe-plugin</artifactId>
+					<version>${maven-surefire-plugin.version}</version>
+				</plugin>
+
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-jar-plugin</artifactId>
+					<version>${maven-jar-plugin.version}</version>
+				</plugin>
+
+				<plugin>
+					<groupId>com.mycila</groupId>
+					<artifactId>license-maven-plugin</artifactId>
+					<version>${license-maven-plugin.version}</version>
+					<configuration>
+						<failIfUnknown>true</failIfUnknown>
+						<header>${maven.multiModuleProjectDirectory}/header.txt</header>
+						<excludes>
+							<exclude>**/*.adoc</exclude>
+							<exclude>**/*.odp</exclude>
+							<exclude>**/*.txt</exclude>
+							<exclude>**/LICENSE.txt</exclude>
+							<exclude>**/LICENSE</exclude>
+							<exclude>**/NOTICE.txt</exclude>
+							<exclude>**/NOTICE</exclude>
+							<exclude>**/README</exclude>
+							<exclude>**/pom.xml.versionsBackup</exclude>
+						</excludes>
+						<mapping>
+							<java>SLASHSTAR_STYLE</java>
+							<properties>CAMEL_PROPERTIES_STYLE</properties>
+							<kt>SLASHSTAR_STYLE</kt>
+						</mapping>
+						<headerDefinitions>
+							<headerDefinition>${maven.multiModuleProjectDirectory}/license-properties-headerdefinition.xml</headerDefinition>
+						</headerDefinitions>
+					</configuration>
+				</plugin>
+			</plugins>
+		</pluginManagement>
+
+		<plugins>
+			<plugin>
+				<groupId>${quarkus.platform.group-id}</groupId>
+				<artifactId>quarkus-maven-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>build</id>
+						<goals>
+							<goal>build</goal>
+						</goals>
+					</execution>
+				</executions>
+			</plugin>
+
+			<plugin>
+				<groupId>net.revelc.code.formatter</groupId>
+				<artifactId>formatter-maven-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>format</id>
+						<goals>
+							<goal>format</goal>
+						</goals>
+						<phase>process-sources</phase>
+					</execution>
+				</executions>
+			</plugin>
+
+			<plugin>
+				<groupId>net.revelc.code</groupId>
+				<artifactId>impsort-maven-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>sort-imports</id>
+						<goals>
+							<goal>sort</goal>
+						</goals>
+						<phase>process-sources</phase>
+					</execution>
+				</executions>
+			</plugin>
+
+			<plugin>
+				<groupId>org.codehaus.gmaven</groupId>
+				<artifactId>groovy-maven-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>sync-k8s-resource-versions</id>
+						<inherited>false</inherited>
+						<goals>
+							<goal>execute</goal>
+						</goals>
+						<phase>process-sources</phase>
+						<configuration>
+							<source>
+                                import java.util.regex.Matcher
```

Comment:
Nitpick - indentation.

Reply:
Nice catch, probably an undue pom editor replace messing up the whole indentation. It's fixed now.